src/icons/folderIcons.ts

```diff
@@ -788,6 +788,10 @@ export const folderIcons: FolderTheme[] = [
         name: 'folder-proto',
         folderNames: ['protobufs'],
       },
+      {
+        name: 'folder-taskfile',
+        folderNames: ['tskf', 'taskfile', '.taskfile', 'Taskfile', '.Taskfile'],
```

Comment:
It might be nice to include the plural versions of these folder names as-per [usage in the Taskfile docs](https://taskfile.dev/usage/#internal-includes) where we conventionally use `taskfiles` as a directory to store included Taskfiles. The dot and titled versions of this also make sense. Not so sure about `tskf`, but 🤷.

Reply:
Hi @pd93, thanks for the feedbacks and i'm glad to help.

- About plural versions: Done
- About `tskf`: Idk too. I didn't think too much about it :sweat_smile:

Reply:
I dont know, maybe it's better to remove `tskf` and keep the taskfile's doc default way.